CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Added
 - Add additional tests and expose private test helper functions in API.
+- Update Bedrock peer dependencies.
```

Comment:
This will be a breaking change right? Since now it requires bedrock-express v6.2.2 and bedrock-server v3.1.0

Reply:
By convention, we have always considered peer deps as "suggestions" which is how it also works in practice.  If there is a mismatch between a peer dependency and the actual version of a package that is used, it only results in a *warning* during `npm i`.

Now, by definition, a breaking change is a change that alters the *public* API surface of a module.  Changing these peer dependency suggestions does not alter the public API surface (as defined by Digital Bazaar).

Changing the *actual* Bedrock module that is installed at the application level can certainly sometimes manifest in a change to the public API surface of the application.

Reply:
While npm (v6 anyway) may only raise a warning if there isn't a match, that doesn't mean that the software will work. I think whether or not this is considered a breaking change should be governed by whether this library will stop working when installed against the old peer dep versions. It seems like it wouldn't in this case.

Note: I also wouldn't consider it a breaking change if this version of the library never actually worked against the old peer deps and this corrects its package.json to list peer deps it would work against.

Reply:
Makes sense, thank you for the explaination.